### PR TITLE
[DOCS] Adds a Jupyter notebook link to the outlier detection example

### PR DESCRIPTION
**Pull request by szabosteve**
This PR adds a link to the outlier detection example that points to a Jupyter notebook on GitHub.

### CODE REVIEW

File: docs/en/stack/ml/df-analytics/ecommerce-outliers.asciidoc [hunk 1]
```diff
@@ -268,3 +268,6 @@ can delete them in {kib} or use the
 {ref}/delete-dfanalytics.html[delete {dfanalytics-job} API]. When
 you delete {transforms} and {dfanalytics-jobs}, the destination indices and
 {kib} index patterns remain.
+
+TIP: https://github.com/blaklaybul/examples/tree/8cff86bce1a05d8bcf1239aab8033490b7c76f2d/Machine%20Learning/Outlier%20Detection/Introduction[If you want to see another example for {oldetection} in a Jupyter notebook, click here.]
```
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/ecommerce-outliers.asciidoc]:
```suggestion
TIP: https://github.com/blaklaybul/examples/tree/8cff86bce1a05d8bcf1239aab8033490b7c76f2d/Machine%20Learning/Outlier%20Detection/Introduction[If you want to see another example of {oldetection} in a Jupyter notebook, click here.]
```

**Comment by blaklaybul** [comment]
wait. the link should be https://github.com/elastic/examples/blob/master/Machine%20Learning/Outlier%20Detection/Introduction/advent-outliers.ipynb

**Comment by blaklaybul** [comment]
and this is the link to the directory on `master`: https://github.com/elastic/examples/tree/master/Machine%20Learning/Outlier%20Detection/Introduction

File: docs/en/stack/ml/df-analytics/ecommerce-outliers.asciidoc [hunk 2]
```diff
@@ -268,3 +268,5 @@ can delete them in {kib} or use the
 {ref}/delete-dfanalytics.html[delete {dfanalytics-job} API]. When
 you delete {transforms} and {dfanalytics-jobs}, the destination indices and
 {kib} index patterns remain.
+
+TIP: https://github.com/blaklaybul/examples/tree/8cff86bce1a05d8bcf1239aab8033490b7c76f2d/Machine%20Learning/Outlier%20Detection/Introduction[If you want to see another example of {oldetection} in a Jupyter notebook, click here.]
```
Comment (szabosteve) [inline docs/en/stack/ml/df-analytics/ecommerce-outliers.asciidoc]:
```suggestion
TIP: https://github.com/elastic/examples/tree/master/Machine%20Learning/Outlier%20Detection/Introduction[If you want to see another example of {oldetection} in a Jupyter notebook, click here.]
```

**Pull request merged**